code/controllers/subsystem/ticker.dm

```diff
@@ -540,6 +540,10 @@ SUBSYSTEM_DEF(ticker)
 			news_message = "The burst of energy released near [station_name()] has been confirmed as merely a test of a new weapon. However, due to an unexpected mechanical error, their communications system has been knocked offline."
 		if(SHUTTLE_HIJACK)
 			news_message = "During routine evacuation procedures, the emergency shuttle of [station_name()] had its navigation protocols corrupted and went off course, but was recovered shortly after."
+		if(GANG_OPERATING)
+			news_message = "[station_name()] has found itself controlled entirely by gangs!"
+		if(GANG_DESTROYED)
+			news_message = "[station_name()] has cleared out all the gangsters infesting the ship!"
```

Comment:
This does not fit in with the rest of the Nanotrasen News Network messages. Muh RP and all that.

Reply:
>ship

station